pkg/render/node.go

```diff
@@ -706,6 +798,46 @@ func (c *nodeComponent) nodeDaemonset(cniCfgMap *corev1.ConfigMap) *appsv1.Daemo
 	return &ds
 }
 
+// vppDaemonset creates the vpp daemonset.
+func (c *nodeComponent) vppDaemonset() *appsv1.DaemonSet {
```

Comment:
I think there is probably a case to made for this DaemonSet and supporting functions being in its own rendered - it's not really part of calico/node. @tmjd WDYT?